DataManagementSystem/Utilities/CSHelpers.py

```diff
@@ -0,0 +1,235 @@
+"""
```

Comment:
Well, this is a matter of taste... Personally I would prefer that systems are consistent and helpers related to a system could be there... Anyway (see above) I would prefer to not group everything into a single helper module that everybody is updating, and therefore nobody is responsible for it...
The most used helper for SE mapping is there:

from DIRAC.Core.Utilities.SiteSEMapping import getSitesForSE

and I don't think this is a proper location, is it?